src/components/desktop/AccountTypeMixin.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  props: {
+    accountType: {
+      type: String,
+      validator: value => ['local', 'remote', 'ledger'].includes(value),
```

Comment:
```suggestion
      validator: value => ['hd-wallet-desktop', 'hd-wallet', 'ledger'].includes(value),
```
why don't use existing account types?